consensus/state_processing/src/per_epoch_processing/base/rewards_and_penalties.rs

```diff
@@ -235,7 +235,7 @@ fn get_inclusion_delay_delta(
         let max_attester_reward = base_reward.safe_sub(proposer_reward)?;
         delta.reward(max_attester_reward.safe_div(inclusion_info.delay)?)?;
 
-        let proposer_index = inclusion_info.proposer_index as usize;
+        let proposer_index: usize = inclusion_info.proposer_index;
```

Comment:
Is the type-hint doing anything here?

```suggestion
        let proposer_index = inclusion_info.proposer_index;
```